VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -2854,6 +2914,56 @@ public void Get_EmergingIssues()
             Assert.Equal("AAAAAAAAAAAAAAAAAAAA", ((DeathRecord)XMLRecords[0]).EmergingIssue20);
         }
 
+        [Fact]
+        public void Set_EntityAxisCodes()
+        {
+            Tuple<string, string, string, string>[] eac = new Tuple<string, string, string, string>[]{Tuple.Create("2", "1", "T273", "Y")};
+            SetterDeathRecord.EntityAxisCauseOfDeath = eac;
+
+            Tuple<string, string, string, string>[] eacGet = SetterDeathRecord.EntityAxisCauseOfDeath;
+            Assert.Equal(1, eacGet.Length);
+            Assert.Equal("2", eacGet[0].Item1);
+            Assert.Equal("1", eacGet[0].Item2);
+            Assert.Equal("T273", eacGet[0].Item3);
```

Comment:
Should this have a decimal point? You may have already mentioned this...

Reply:
yes, fixed